README.md

```diff
@@ -1,222 +1,73 @@
-
-Use the Sass `@import` directive to include a component’s styles. For example, in `my-project/styles/stylesheet.scss`, I can import the Grid component like so:
-
-```scss
-@import '../bower_components/mobify-stencil/dist/components/grid/grid';
-```
-
-Always import Stencil components *after* your Sass variables and *before* any of your own components. This ensures that Stencil receives the values you want for any of its configurable variables while allowing you to reliably build on what it provides.
-
-
-## Using a Stencil component
-
-Stencil components are like any component you build yourself, except (as with all dependecies) you should never modify them directly. You can *configure* a Stencil component’s CSS by overriding its configurable variables in your own stylesheets. You can also *extend* components by styling them as you see fit and creating new variations on existing components. Feel free to override Stencil styles as well, although Stencil tries to be as minimal as possible so you shouldn’t often need to.
-
-One of Stencil’s advantages is that you can work more with HTML and write less custom CSS. Once your theming is in place and you’ve build any custom components you need, you might find you can compose whole screens of UI by simply writing HTML implementing the markup patterns (structure and classes) defined by the component.
+Install with [Grunt](http://gruntjs.com/):
 
-Note that Stencil’s components are designed to be robust. As long as you *structure* your markup as according to a component’s example markup and apply the correct classes, you should be free to use the most appropriate, semantic HTML elements for your use case.
-
-
-### Configuring and extending
-
-Components are customizable in two ways: *configuration* and *extension*. Configure a component by setting a component’s variables *before* you import the component, and extend by adding/overriding styles or adding new modifiers *after* importing the component. The order is important, because each component will use its default values for variables unless you provide your own. For example, assume you have the following in your main stylesheet:
-
-```scss
-@import 'variables'; // Do this first, so Stencil is aware of your settings.
-@import '../bower_components/mobify-stencil/dist/components/button/button'; // Import Stencil components before yours.
-@import 'my-components/button'; // Add styles to Stencil components after importing them.
 ```
-
-The contents of these files might look like this:
-
-```scss
-// _variables.scss
-
-// Configure some variables for Stencil’s button:
-$button-padding: 5px 10px;
-$button-border-radius: 5px;
+grunt component:install:utils
 ```
 
-```scss
-// my-components/_button.scss
+Once installed, the Stencil Utilities live in your project’s `app/bower_components` directory. Individual utilities will provide a stylesheet, including example markup and (optionally) JavaScript. To learn more, visit the [dev centre documentaton](#).
```

Comment:
Utils in their present form should not have JS; even an expanded concept of them probably shouldn’t. I’d take this statement out since for now it will only serve to confuse things.